src/app-layout/__tests__/runtime-drawers.test.tsx

```diff
@@ -72,6 +75,65 @@ describeEachAppLayout(({ theme, size }) => {
     expect(wrapper.findDrawersTriggers()).toHaveLength(2);
   });
 
+  test.only('update rendered drawers via runtime config', async () => {
```

Comment:
This line accidentally skipped other tests

```suggestion
  test('update rendered drawers via runtime config', async () => {
```

Reply:
updated.